sensing/pointcloud_preprocessor/docs/ring-outlier-filter.md

```diff
@@ -27,7 +27,7 @@ This implementation inherits `pointcloud_preprocessor::Filter` class, please ref
 | `distance_ratio`          | double  | 1.03          |             |
 | `object_length_threshold` | double  | 0.1           |             |
 | `num_points_threshold`    | int     | 4             |             |
-| `max_rings_num`           | uint_16 | 128           |             |
+| `max_ring_index`          | uint_16 | 128           |             |
```

Comment:
@sykwer Could you please use velodyne 128 as an example here and write more details in the description?
Also, is 127 correct for index?
```suggestion
| `max_ring_index`          | uint_16 | 127           |             |
```

Reply:
Sure.

> In a project using Autoware, max_rings_num was set to 128 (i.e. the range of ring index was set to be 0~127), but actually, the maximum value of ring index was 128.
In the case described above, `128` is correct for `max_ring_index`, so I'd like to set the default value to be `128`.

Reply:
it looks like 128 is better because it is vls128 and i think implementation should be corrected according to the parameter.

Reply:
>it looks like 128 is better because it is vls128 and i think implementation should be corrected according to the parameter.

Agree with you :+1: